### PR TITLE
Create 2017-10-30-Gabon-bioinformatics

### PR DESCRIPTION
**Pull request by anelda**

### CODE REVIEW

File: _posts/2017/10/2017-10-30-Gabon-bioinformatics [hunk 1]
```diff
@@ -7,23 +7,59 @@ time: "09:00:00"
...
+In March 2017 Jason Williams introduced Nicola Anthony to the South African Carpentry community to share some lessons learned 
+about running computing workshops in Africa. During preliminary conversations we introduced the UNO/USTM teams to principles of 
+the Software and Data Carpentry workshops such as live coding, sticky notes for getting help or indicating progress, 
+and a variety of other practical things. 
+
+The workshop ran from 3 - 8 July 2017 and it was a great privilege that one of our South African instructors, Samar Elsheikh, 
+was able to join the team in Franceville for the entire event. Topics that we covered included: data organization and spreadsheets, 
+working in the command line, data visualization in R, next generation sequencing methods, processing restriction site associated 
+DNA (RAD) sequencing, detecting loci under selection and geospatial modeling of genomic variation. A more detailed copy of 
+the program is available here: LINK.
```
Comment (jduckles) [inline _posts/2017/10/2017-10-30-Gabon-bioinformatics:47]: LINK is still missing?

**Comment by jduckles** [comment]
@anelda, was this taught from SWC or DC lessons? If DC, it might be good to cross-post to DC. If neither, here is probably as good as anywhere else.

**Comment by weaverbel** [comment]
Also @anelda and @jduckles I have concerns about the title - does Git render the post if it includes capital letters? ie Gabon should be gabon??

**Comment by anelda** [comment]
@jduckles I will confirm, but think both SWC (Shell) and DC (Excel, R). @weaverbel I am waiting for the link for the schedule after which I will change the name to make it small letters only. Thanks for your feedback!

**Comment by jduckles** [comment]
@anelda great, just let me know when you're ready to merge and I'll push the magic button! Thank you for the contribution!

**Comment by jduckles** [comment]
@anelda just a ping to see when you'd like to have this merged.

**Comment by anelda** [comment]
Have been replaced by #884

**Pull request closed without merging**